tests/foreman/maintain/test_upgrade.py

```diff
@@ -146,11 +147,16 @@ def test_negative_pre_upgrade_tuning_profile_check(request, custom_host):
     )
     custom_host.download_repofile(product='satellite', release=last_y_stream)
     custom_host.execute('dnf -y module enable satellite:el8 && dnf -y install satellite')
-    # Install without system checks to get around installer checks
+    # Install with development tuning profile to get around installer checks
```

Comment:
if you are changing this test to use `development` profile the you should also handle tests parametrization ` ids=['default', 'medium'],` 

and all asserts for `{profile}`

Reply:
Funny part is green PRT for this NEGATIVE test - failing on something else :D

Reply:
We are only using the development profile to get around installer checks for system requirements like RAM and CPU cores. We have to do this in >= 6.14 because `--disable-system-checks` was removed. This test is still designed to make sure the default and medium profiles enforce their requirements upon an upgrade check.